data/moves.js

```diff
@@ -17186,7 +17186,7 @@ exports.BattleMovedex = {
 		effect: {
 			duration: 2,
 			onStart: function (target) {
-				this.add('-start', target, 'move: Throat Chop');
+				this.add('-start', target, 'move: Throat Chop', '[silent]');
```

Comment:
Because I copypasted Taunt, which also does this, as well as most volatiles that are effects of a move.